make/source.mk

```diff
@@ -93,7 +93,7 @@ COMMON_SRC = \
             flight/gps_rescue.c \
             flight/gyroanalyse.c \
             flight/imu.c \
-            flight/interpolated_setpoint.c \
+            flight/ff.c \
```

Comment:
'ff' is not understandable. When used alone, this should really be 'feedforward'.

Reply:
Thanks.  I wasn't sure about that.  I wavered :-)
ff is nice and simple and easy to type.
rc, imu, rc, osd etc are all short abbreviations for longer names.  
we use ff extensively now in the CLI, as an abbreviation, and I'm hoping to be consistent in its application in the code base.
So yes, feedforward.c could be used as a longer and more transparent name than ff.c, and it's not so long.
Let's think about it a day or so, happy to use either.